From: Colin Ian King <colin.king@canonical.com>
Date: Sat, 5 Aug 2017 14:46:35 +0100
Subject: net: hns3: fix spelling mistake: "capabilty" -> "capability"
Patch-mainline: v4.14-rc1
Git-commit: e00e21979dc8a821840f23a8f1d62bd823755192
References: bsc#1104353 FATE#326415

Trivial fix to spelling mistake in dev_err error message and also
split overly long line to avoid a checkpatch warning.

Signed-off-by: Colin Ian King <colin.king@canonical.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Thomas Bogendoerfer <tbogendoerfer@suse.de>
---
 drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_main.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_main.c
+++ b/drivers/net/ethernet/hisilicon/hns3/hns3pf/hclge_main.c
@@ -4089,7 +4089,8 @@ static int hclge_init_ae_dev(struct hnae
 
 	ret = hclge_get_cap(hdev);
 	if (ret) {
-		dev_err(&pdev->dev, "get hw capabilty error, ret = %d.\n", ret);
+		dev_err(&pdev->dev, "get hw capability error, ret = %d.\n",
+			ret);
 		return ret;
 	}
 
